src/containers/Desk.js

```diff
@@ -125,114 +95,108 @@ class Desk extends Component {
     this.incrementViews(letter)
   }
 
-  incrementViews = letter => {
-    fetch(LETTERS_URL + `/${letter.id}`, {
-      method: 'PATCH',
+  postSeen = letterId => {
+    const { accountId } = this.props
+
+    fetch(SEENS_URL, {
+      method: 'POST',
       headers: {
         'Content-Type': 'application/json',
         Accept: 'application/json'
       },
       body: JSON.stringify({
-        num_views: letter.num_views + 1
+        account_id: accountId,
+        letter_id: letterId
       })
     })
-      .then(res => res.json())
-      .then(console.log)
   }
 
-  incrementResponses = letter => {
+  incrementViews = letter => {
     fetch(LETTERS_URL + `/${letter.id}`, {
       method: 'PATCH',
       headers: {
         'Content-Type': 'application/json',
         Accept: 'application/json'
       },
       body: JSON.stringify({
-        num_responses: letter.num_responses + 1
+        num_views: letter.num_views + 1
       })
     })
-      .then(res => res.json())
-      .then(console.log)
   }
 
-  renderRead = letter => {
+  renderWrite = () => {
+    const { accountId, icon } = this.props
     this.stopPlanes()
-    return <Read letter={letter} handleClick={this.handleRespondClick} />
+    return <Write accountId={accountId} icon={icon} setDesk={this.setDesk} />
   }
 
-  handleRespondClick = () => {
-    this.setState({
-      isJournal: false,
-      isRead: true,
-      isWrite: true
-    })
+  renderRead = letter => {
+    this.stopPlanes()
+    return <Read letter={letter} setDesk={this.setDesk} />
   }
 
   renderCreateResponse = letter => {
+    const { accountId, icon } = this.props
+
     return (
       <CreateResponse
-        accountId={this.props.accountId}
-        icon={this.props.icon}
+        accountId={accountId}
+        icon={icon}
         letter={letter}
         isRead={true}
         isWrite={true}
-        handleCloseClick={this.clearDesk}
+        setDesk={this.setDesk}
         incrementResponses={this.incrementResponses}
       />
     )
   }
 
-  handleJournalClick = () => {
-    this.setState({
-      isJournal: true,
-      isRead: false,
-      isWrite: false
+  incrementResponses = letter => {
+    fetch(LETTERS_URL + `/${letter.id}`, {
```

Comment:
```suggestion
    fetch(${LETTERS_URL}/${letter.id}`, {
```
Concat + Interpolation